service/src/main/kotlin/fi/espoo/evaka/attendance/ChildAttendanceController.kt

```diff
@@ -114,6 +117,7 @@ class ChildAttendanceController(
                         )
                     }
                 } catch (e: Exception) {
+                    logger.error(e.toString())
```

Comment:
Kind of, I added this as `mapPSQLException` throws the error message away and only uses the SQL exception cause for the message, so it's not possible to know why an update/insert failed (e.g., which constraint). Do you think this could be useful in other environments, or should I remove this?

Reply:
I think it's an useful idea, and we could use the standard cause mechanism of exceptions for it:

```
fun mapPSQLException(e: Exception): Exception {
    ...
    Conflict("Unique or exclusion constraint violation in database", cause = e)
    ...
}
```

This is probably best done in a separate pull request, since we could add the same mechanism in all of the Conflict/Unauthorized/etc types, and might need to fix some logging too if we only log the message and don't pass the exception to the logging system.

Reply:
That sounds like a good idea; I've removed the logging additions here 👍